simbad/rotsearch/amore_search.py

```diff
@@ -40,12 +36,13 @@ class AmoreRotationSearch(object):
         The path to the working directory
     max_to_keep : int
         The maximum number of results to keep [default: 20]
+    search_results : list
+        The search results
 
     Examples
     --------
     >>> from simbad.rotsearch.amore_search import AmoreRotationSearch
     >>> rotation_search = AmoreRotationSearch('<amore_exe>', '<mtz>', '<work_dir>', '<max_to_keep>')
-    >>> rotation_search.sortfun()
     >>> rotation_search.run_pdb(
```

Comment:
This example needs updating to reflect the name change from run_pdb to run

Reply:
Addressed in c01b3ab